src/mongo/db/auth/external/cyrus_sasl_server_session.cpp

```diff
@@ -81,15 +81,16 @@ struct gss_result {
     OM_uint32 major = 0;
     OM_uint32 minor = 0;
     void check(const char* loc) const {
-        if (major != GSS_S_COMPLETE)
+        if (major != GSS_S_COMPLETE) {
             throw std::runtime_error(fmt::format("{} error: major: {}; minor: {}",
                                                  loc, major, minor));
+        }
     }
 };
 
 class auto_gss_name_t {
 public:
-    auto_gss_name_t() : _v(nullptr) {}
+    auto_gss_name_t() = default;
```

Comment:
Can we remove the default constructor declaration/definition and rely on the compiler automatically generating one?